app/routes/events/components/EventAdministrate/RegistrationTables.js

```diff
@@ -61,13 +61,21 @@ export class RegisteredTable extends Component<Props> {
       {
         title: 'Status',
         dataIndex: 'pool',
-        render: pool => (
+        render: (pool, registration) => (
           <TooltipIcon
-            content={pool ? 'Påmeldt' : 'Venteliste'}
+            content={
+              registration.adminRegistrationReason !== ''
+                ? 'Adminpåmeldt: ' + registration.adminRegistrationReason
+                : pool
+                ? 'Påmeldt'
+                : 'Venteliste'
```

Comment:
Nested ternary is not very readable 😞

Reply:
True, but couldn't be bothered as the linter wasn't complaining. #todo? 

We should add `no-nested-ternary` to our eslint config if we don't want that in the future :shrug: